.github/workflows/deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and push container images
+
+on:
+  push:
+    branches:
+      - deploy/*
+
+jobs:
+  build-and-push-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to the container registry
+        uses: docker/login-action@v1
+        with:
+          registry: gcr.io
+          username: _json_key
```

Comment:
This looks really weird. Is it correct?

Reply:
Corrent, it requires copying over the `GCR_JSON_KEY` from the https://github.com/radicle-dev/radicle-client-services/ repo before we first use this job to deploy though

Reply:
Hmf ok.